README.md

```diff
@@ -86,6 +96,46 @@ dbt_fullstory:
       query_tag: [fullstory_dbt]
 ```
 
+#### Redshift
+```yaml
+dbt_fullstory:
+  target: prod
+  outputs:
+    prod:
+      type: redshift
+      cluster_id: my-cluster-id
+      method: iam
+      host: my-cluster-id.12345678910.us-east-1.redshift.amazonaws.com
+      port: 5439
+      user: serverlessuser
```

Comment:
why does serverlessuser show up here?

Reply:
Copy pasta, changed it to `admin` to represent the default user for classic redshift.